ui/pages/confirmations/hooks/alerts/transactions/useNonContractAddressAlerts.test.ts

```diff
@@ -216,6 +218,46 @@ describe('useNonContractAddressAlerts', () => {
     ).toEqual([]);
   });
 
+  it('returns no alert for authorization request', async () => {
+    const authorizationList = [{ address: '0x123' as Hex }];
+    const transaction = genUnapprovedContractInteractionConfirmation({
+      authorizationList,
+    });
+
+    useContextMock.mockImplementation((context) => {
+      if (context === ConfirmContext) {
+        return { currentConfirmation: transaction };
+      } else if (context === I18nContext) {
+        return (translationKey: string) => translationKey;
+      }
+      return undefined;
+    });
+
+    useSelectorMock.mockImplementation((selector) => {
```

Comment:
Not introduced by this PR, but worth refactoring these tests in future to avoid selector and context mocking, as we aim to use real state and real contexts to keep the tests more resilient.